packages/front-end/components/Features/EditAttributesModal.tsx

```diff
@@ -64,6 +65,37 @@ export default function EditAttributesModal({ close }: { close: () => void }) {
         </p>
       )}
       <div className="form-inline">
+        <div className="pb-2 d-flex align-items-center">
+          <Toggle
+            value={attributeSchema.fields.some(
+              (attribute) => attribute.property === "current_datetime"
+            )}
+            setValue={() => {
+              if (
+                !attributeSchema.fields.some(
+                  (attribute) => attribute.property === "current_datetime"
+                )
+              ) {
+                attributeSchema.append({
+                  property: "current_datetime",
+                  datatype: "date",
+                });
+              } else {
+                attributeSchema.remove(
+                  attributeSchema.fields.findIndex(
+                    (attribute) => attribute.property === "current_datetime"
+                  )
+                );
+              }
+            }}
+            id="full-stats"
+            label="Show Full Stats"
+          />
+          Enable date/time based feature rules
```

Comment:
To me this feels a bit unintuitive, and perhaps also easy for the user to corrupt the magic "current_datetime" id. I wonder if perhaps date/time attributes should always be present (hard-coded) in the feature targeting modal.

Reply:
Definitely agree this it is a bit of a weird flow. The current plan is this is a stop-gap until we can update all of the SDK's to automatically set a `current_datetime` attribute when instantiated. Once that is done, we can remove this and make it automatic for all organizations.

Another alternative that we discussed was building a way to identify which SDK the organization was using, and only add the `current_datetime` attribute/operator to the feature rule modal if the SDK they're using supports. 

Both of those require some work.

I'm open to other suggestions, and will spend a few minutes brainstorming today some potential alternatives. Thank you!

Reply:
Update: Bryce and I spoke and discussed a treatment that would remove all of the updated code within `EditAttributesModal` and instead, just hard-code an option to build a rule condition based on `current_datetime` within the `ConditionInput` component along with a warning message that this is only supported by the JS SDK (atleast, initially).

@jdorn - Curious to hear your thoughts - I recall going the route with the toggle was meant to be a bit of an explicit action in order to prevent an organization from creating a datetime based rule when the SDK the org is using doesn't yet support that functionality.

Reply:
I'm fine with this approach.  I think at launch we can have support for Ruby, PHP, Python, Javascript, and React pretty easily.  Those are all less than 5 lines of code to implement.